common/src/main/scala/org/apache/celeborn/common/util/CelebornHadoopUtils.scala

```diff
@@ -18,12 +18,23 @@
 package org.apache.celeborn.common.util
 
 import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
 
 import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.internal.Logging
 
-object CelebornHadoopUtils {
+object CelebornHadoopUtils extends Logging {
   private[celeborn] def newConfiguration(conf: CelebornConf): Configuration = {
     val hadoopConf = new Configuration()
+    if (!conf.hdfsDir.isEmpty) {
+      val path = new Path(conf.hdfsDir)
+      val scheme = path.toUri.getScheme
+      val disableCacheName = String.format("fs.%s.impl.disable.cache", scheme)
+      hadoopConf.set("dfs.replication", "2")
+      hadoopConf.set(disableCacheName, "false")
+      logInfo(s"Celeborn will ignore cluster settings $disableCacheName and " +
+        "set it to false")
```

Comment:
this may be a good place to print the overwrite behavior.

Celeborn overrides some HDFS settings defined in Hadoop configuration files, including overriding '$disableCacheName=false' and 'dfs.replication=2'. It can be overridden again in the Celeborn configuration with the additional prefix 'celeborn.hadoop.'

Reply:
To un-noisy users, you may want to add a flag to ensure this message only prints one time.

Reply:
OK